### ISSUE TITLE
How to save notes from previous comm calls

### ISSUE THREAD

**Issue by maelle** [ropensci/commcalls#25]
Etherpad is going to disappear https://wiki.mozilla.org/Etherpad

E.g. https://public.etherpad-mozilla.org/p/ropensci.comm-call-04-15

**Comment by karthik** [comment]
Google docs seems best going forward. At least when they sunset there is a takeaway plan.

**Comment by maelle** [comment]
Yes but we need to do sthg about the existing Etherpad notes I guess 😢

**Comment by karthik** [comment]
Just copy them over to new Google docs?
- Then save all Google docs in the same folder.
- Set permissions to view only so that people can read but not edit old calls.
- For all new calls, keep doc globally editable till the meeting, then Stef can clean up, and finally it also goes into read-only.
